### PR TITLE
Release/1.2.0

### PR DESCRIPTION
**Pull request by yzha645**
This PR bumps chart version to 1.2.0 for all products.
For Bamboo and Bamboo agent, 1.1.0 is skipped to keep consistent with the other products.

### CODE REVIEW

File: src/main/charts/jira/Changelog.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+**Release date:** 2022-02-10
+
+![AppVersion: 8.20.5-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=8.20.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* DCD-1452: Updated appVersion to the latest product LTS version. (#378)
+* Improvements on documentation (#370, #357)
```
Comment (nghazali) [inline src/main/charts/jira/Changelog.md:13]:
```suggestion
* Improvements on [documentation](https://github.com/atlassian/data-center-helm-charts/) (#370, #357)
```

File: src/main/charts/crowd/Changelog.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+**Release date:** 2022-02-10
+
+![AppVersion: 4.4.0-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=4.4.0-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+Crowd (1.1.0 -> 1.2.0)
+* Added missing JVM_SUPPORT_RECOMMENDED_ARGS env (#376)
+* Improvements on documentation (#370, #357)
```
Comment (nghazali) [inline src/main/charts/crowd/Changelog.md:14]:
```suggestion
* Improvements on [documentation](https://github.com/atlassian/data-center-helm-charts/) (#370, #357)
```

File: src/main/charts/confluence/Changelog.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+
+**Release date:** 2022-02-10
+
+![AppVersion: 7.13.4-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=7.13.2-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* DCD-1452: Updated appVersion to the latest product LTS version. (#378)
+* Improvements on documentation (#370, #357)
```
Comment (nghazali) [inline src/main/charts/confluence/Changelog.md:13]:
```suggestion
* Improvements on [documentation](https://github.com/atlassian/data-center-helm-charts/) (#370, #357)
```

File: src/main/charts/bitbucket/Changelog.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+
+![AppVersion: 7.17.5-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=7.17.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* Use volumeName only if explicitly defined (#372)
+* Fixed templating of PLUGIN_SSH_BASEURL with custom port (#371)
+* DCD-1452: Updated appVersion to the latest product LTS version. (#378)
+* Improvements on documentation (#370, #357)
```
Comment (nghazali) [inline src/main/charts/bitbucket/Changelog.md:15]:
```suggestion
* Improvements on [documentation](https://github.com/atlassian/data-center-helm-charts/) (#370, #357)
```

File: src/main/charts/bamboo-agent/Changelog.md [hunk 5]
```diff
@@ -1,5 +1,21 @@
...
+**Release date:** 2022-02-10
+
+![AppVersion: 8.1.2-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=8.1.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* DCD-1452: Updated appVersion to the latest product LTS version. (#378)
+* Added end-to-end test for Bamboo Helm chart using Terraform (#375)
+* Improvements on documentation (#370)
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:14]:
```suggestion
* Improvements on [documentation](https://github.com/atlassian/data-center-helm-charts/) (#370)
```

**Review by nghazali — APPROVED** [review]

**Comment by nghazali** [comment]
Added some suggestions for link to documentation to be consistent with the existing content in Change log file.

File: src/main/charts/bamboo/Changelog.md [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+
+**Release date:** 2022-02-10
+
+![AppVersion: 8.1.2-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=8.1.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* DCD-1452: Updated appVersion to the latest product LTS version. (#378)
+* Added end-to-end test for Bamboo Helm chart using Terraform (#375)
+* Improvements on documentation (#370)
```
Comment (yzha645) [inline src/main/charts/bamboo/Changelog.md:13]:
```suggestion
* Improvements on [documentation](https://github.com/atlassian/data-center-helm-charts/) (#370)
```

File: src/main/charts/bamboo-agent/Changelog.md [hunk 7]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```
Comment (nanux) [inline src/main/charts/bamboo-agent/Changelog.md:5]:
```suggestion
**Release date:** 2022-02-11
```

Can we update this for all products (if we are releasing today).

**Review by nanux — APPROVED** [review]

File: src/main/charts/bamboo/Changelog.md [hunk 8]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```
Comment (yzha645) [inline src/main/charts/bamboo/Changelog.md:5]:
```suggestion
**Release date:** 2022-02-11
```

File: src/main/charts/bitbucket/Changelog.md [hunk 9]
```diff
@@ -1,5 +1,29 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```
Comment (yzha645) [inline src/main/charts/bitbucket/Changelog.md:5]:
```suggestion
**Release date:** 2022-02-11
```

File: src/main/charts/confluence/Changelog.md [hunk 10]
```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```
Comment (yzha645) [inline src/main/charts/confluence/Changelog.md:5]:
```suggestion
**Release date:** 2022-02-11
```

File: src/main/charts/crowd/Changelog.md [hunk 11]
```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```
Comment (yzha645) [inline src/main/charts/crowd/Changelog.md:5]:
```suggestion
**Release date:** 2022-02-11
```

File: src/main/charts/jira/Changelog.md [hunk 12]
```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```
Comment (yzha645) [inline src/main/charts/jira/Changelog.md:5]:
```suggestion
**Release date:** 2022-02-11
```

File: src/main/charts/bamboo-agent/Changelog.md (hunk 7 above)
Comment (nghazali) [reply src/main/charts/bamboo-agent/Changelog.md:5]: If the release go to Monday let's update the release date then before the release.

Comment (yzha645) [reply src/main/charts/bamboo-agent/Changelog.md]: Updated~

**Pull request merged**